Client/src/main/java/org/projectcardboard/client/models/game/availableactions/AvailableActions.java

```diff
@@ -242,10 +243,31 @@ private boolean isTroopProvoked(GameMap gameMap, Player player, Troop troop) {
         return false;
     }
 
+    public boolean canCastSpellOnSquare(GameMap gameMap, Player player, Card card, int row, int column) {
```

Comment:
I think this name of this function is misleading.  

One day we may add cards that target tiles (eg shadow creep).  The current implementation checks if we can target a 'unit' on a given square.

Reply:
This function also checks for global spells, not just units. If I didn't interpret your comment correctly, what would be the new name of this function?